src/private/patient/registerVisit.html

```diff
@@ -44,8 +45,8 @@ <h4 class="center" style="padding: 12px 24px 32px 24px;">Nowa wizyta</h4>
                     <div class="input-field col col s12 m6 l6">
                         <select id="select-duration">
                             <option value="" disabled selected>Wybierz czas trwania</option>
-                            <option value="1">1 godzina</option>
-                            <option value="2">2 godziny</option>
+                            <option value="1">10 minut (wystawienie recepty)</option>
+                            <option value="2">30 minut (konsultacja)</option>
```

Comment:
Zmieniając to zmieniasz jedynie co się wyświetla w tekście. W skrypcie wykorzystywane są przypisane tu 'value'.
Poniżej masz snippet, gdzie biorę te value, i na jego podstawie określam wartość w sekundach, która zostanie użyta.
registerVisitScript.js
`    let appointmentDuration = selectDuration.options[selectDuration.selectedIndex].value;
    // Appointment duration should be presented in seconds
    if (appointmentDuration == 1) {
        submittedData.duration = 3600;
    } else if (appointmentDuration == 2) {
        submittedData.duration = 7200;
    } else submittedData.duration = 0;`